apps/healthlocker/web/templates/tracker/tracking_data.html.eex

```diff
@@ -16,11 +16,14 @@
             <%= img_tag("/images/Toolkit_sleep.png", class: "w-90 tracker-icon") %>
           </div>
           <div class="di w-80 w-90-m w-90-l self-end pb2-m pb2-l pl3-m pl3-l">
-            <%= if data.sleep_data.hours_slept == "0" do %>
-             <%= "0 hours" %>
-            <% else %>
-             <%= format_average_sleep(String.to_float(data.sleep_data.hours_slept)) %>
-            <% end %>
+            <p class="mb0 pr3-m pr3-l di-m di-l">
+              <strong>Sleep: </strong>
+              <%= if data.sleep_data.hours_slept == "0" or data.sleep_data.hours_slept == "0.0" do %>
```

Comment:
:+1: the ```or``` operator only accept Boolean compare to ```||``` which can accept any values: https://elixir-examples.github.io/examples/boolean-operators